tests/e2e/api/controllers/users.js

```diff
@@ -274,6 +314,42 @@ describe('Users API', () => {
 
     });
 
+    it('should allow to update the admin password and login successfully', async () => {
+      const newPassword = 'medic.456';
+      const otherAdmin = {
+        username: 'admin2',
+        password: 'medic.123',
+        roles: [ 'admin' ],
+        contact: { name: 'Philip' },
+        place: newPlaceId,
+      };
+      await utils.createUsers([ otherAdmin ]);
+      // Set admin user's password in CouchDB config.
+      await utils.request({
+        port: constants.COUCH_PORT,
+        method: 'PUT',
+        path: `/_node/${constants.COUCH_NODE_NAME}/_config/admins/${otherAdmin.username}`,
+        body: `"${otherAdmin.password}"`,
+      });
+
+      // Update password with new value.
+      const userResponse = await utils
+        .request({
+          path: `/api/v1/users/${otherAdmin.username}`,
+          method: 'POST',
+          body: { password: newPassword }
+        })
+        .catch(() => chai.assert.fail('Should not throw error'));
+
+      chai.expect(userResponse.user).to.not.be.undefined;
+      chai.expect(userResponse.user.id).to.equal('org.couchdb.user:admin2');
+      chai.expect(userResponse['user-settings']).to.not.be.undefined;
+      chai.expect(userResponse['user-settings'].id).to.equal('org.couchdb.user:admin2');
+      // Old password shouldn't work.
+      expectPasswordLoginToFail(otherAdmin);
+      expectPasswordLoginToWork({ username: otherAdmin.username, password: newPassword });
```

Comment:
These two are promises that we must wait for.

Reply:
Done